src/luascript.cpp

```diff
@@ -10667,6 +10676,19 @@ int LuaScriptInterface::luaItemTypeCreate(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaItemTypeIsBlocking(lua_State* L)
+{
+	// itemType:isBlocking()
+	const ItemType* itemType = getUserdata<const ItemType>(L, 1);
+	if (itemType) {
+		pushBoolean(L, itemType->blockProjectile || itemType->blockSolid);
+	}
```

Comment:
I'm not seeing the difference between this and every other instance this occurs in functions throughout this file... what did I do wrong here? 10685 has a bracket 10686 has else {